app/uk/gov/hmrc/helptosavefrontend/views/email/accountholder/check_your_email.scala.html

```diff
@@ -18,52 +18,52 @@
-      <li><a href=@{frontendAppConfig.nsiManageAccountUrl}>@messages("hts.global.breadcrumb.account.home")</a></li>
-    </ol>
-  </div>
+@beforeContentBlock = {
+    @viewHelpers.govukBreadcrumbs(Breadcrumbs(
+        items = Seq(
+        BreadcrumbsItem(
+        content = Text(messages("hts.global.breadcrumb.account.home")),
+        href = Some(frontendAppConfig.nsiManageAccountUrl)
+        ))
+    ))
 }
 
 @title = @{
   Messages("hts.email-verification.existing-user.check-your-email.title.h1")
 }
 
-@main(
+@layout(
   title = title  + " " + Messages("hts.global.title.section.email"),
-  sidebarLinks = None,
-  bodyClasses = None,
-  serviceInfoContent = serviceInfoContent) {
+        beforeContent = Some(beforeContentBlock)) {
 
-    <h1 class="heading-large">@title</h1>
-    <p>@Html(Messages("hts.email-verification.existing-user.check-your-email.we-have-sent-mail", email, routes.AccountHolderController.getUpdateYourEmailAddress.url))</p>
+    <h1 class="govuk-heading-l">@title</h1>
+    <p class="govuk-body">@Html(Messages("hts.email-verification.existing-user.check-your-email.we-have-sent-mail", email, routes.AccountHolderController.getUpdateYourEmailAddress.url))</p>
     
-    <h2 class="heading-medium">@messages("hts.email-verification.existing-user.check-your-email.what-next")</h2>
-    <p class="content">@Html(Messages("hts.email-verification.existing-user.check-your-email.content1"))</p>
+    <h2 class="govuk-heading-m">@messages("hts.email-verification.existing-user.check-your-email.what-next")</h2>
+    <p class="govuk-body">@Html(Messages("hts.email-verification.existing-user.check-your-email.content1"))</p>
 
-    <h2 class="heading-medium">@messages("hts.email-verification.existing-user.check-your-email.h3")</h2>
+    <h2 class="govuk-heading-m">@messages("hts.email-verification.existing-user.check-your-email.h3")</h2>
 
-    @{
-      helpers.form(routes.AccountHolderController.onSubmit)(
-        Html(
-          s"""
-          |<input type="hidden" name="new-email-address" value="$email">
-          |<p>${Messages("hts.email-verification.existing-user.check-your-email.resend-1")}
-          |${button('_modifierClass -> "button button--link soft push--right", 'role -> "link", 'id → "resend-verification")(Html(messages("hts.email-verification.existing-user.check-your-email.resend-button-text")))}
-          |${Messages("hts.email-verification.existing-user.check-your-email.resend-2")}</p>
-          |""".stripMargin
-        )
-      )
-    }
-    
-    <p class="subsection--wide push--bottom"><a href=@{frontendAppConfig.nsiManageAccountUrl} class="button" role="button">@messages("hts.global.back-to-account")</a></p>
+@viewHelpers.form(uk.gov.hmrc.helptosavefrontend.controllers.routes.AccountHolderController.onSubmit) {
+<p class="govuk-body">@Messages("hts.email-verification.existing-user.check-your-email.resend-1")
 
-  
+    @viewHelpers.govukButton(
+    Button(
+    attributes = Map("id" -> "resend-verification"),
```

Comment:
to reproduce what we had before upgrading this button should have an extra class added to make it secondary button instead of a green, call-to-action button e.g. `classes="govuk-button--secondary",`